src/main/java/pl/nitl/employeeManager/models/AddressType.java

```diff
@@ -0,0 +1,8 @@
+package pl.nitl.employeeManager.models;
+
+public enum AddressType {
+
+    ZAMIESZKANIA,
```

Comment:
troche to kluje w oczy te polskie typy, wez sobie wymysl jakies angielskie, Postal, Invoiceing, current etc